package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/http-transport-emf-stats",
-  "version": "1.1.1",
+  "version": "2.0.0-ALPHA-2",
```

Comment:
I used the ALPHA label in order to publish this package on NPM early on a tag and test it directly from an environment. Once the PR is reviewed this will be updated to just `2.0.0` 👍

Reply:
👍